cmd/influx_inspect/export/export.go

```diff
@@ -56,17 +56,21 @@ func NewCommand() *Command {
 func (cmd *Command) Run(args ...string) error {
 	var start, end string
 	fs := flag.NewFlagSet("export", flag.ExitOnError)
-	fs.StringVar(&cmd.dataDir, "datadir", os.Getenv("HOME")+"/.influxdb/data", "Data storage path. [$HOME/.influxdb/data]")
-	fs.StringVar(&cmd.walDir, "waldir", os.Getenv("HOME")+"/.influxdb/wal", "Wal storage path. [$HOME/.influxdb/wal]")
+	fs.StringVar(&cmd.dataDir, "datadir", os.Getenv("HOME")+"/.influxdb/data", "Data storage path.")
```

Comment:
What do you think about cribbing something like [this](https://github.com/mitchellh/go-homedir/blob/master/homedir.go#L120-L137) for slightly better Windows support?